### PR TITLE
Feature/166002062 add question mark tooltip

### PR DESCRIPTION
**Pull request by lingyun1010**
I add a tooltip based on Pablo's information text and correct the experiment description link in the table content. Please have a review. Thanks.

### CODE REVIEW

File: src/TableContent.js [hunk 1]
```diff
@@ -2,8 +2,17 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { Table } from 'evergreen-ui'
 import URI from 'urijs'
+import ReactTooltip from 'react-tooltip'
 
 import tableHeaderCells from './tableHeaderCells'
+import TooltipIcon from './TooltipIcon'
+
+const downloadTooltip = `<ul>
```
Comment (monicajianu) [inline src/TableContent.js]: I can't remember the discussion in the planning meeting, but did we agree we'll pass these in as props? This tooltip won't be exactly the same for bulk Atlas.

File: src/TableContent.js [hunk 2]
```diff
@@ -24,16 +33,19 @@ const TableContent = ({tableHeader, searchedColumnIndex, searchQuery, orderedCol
             enableDownload && <Table.TextHeaderCell className={`downloadHeader`} flexBasis={100} flexShrink={100} flexGrow={100}>
               {
                 checkedRows.length > 0 ?
-                  <a href={URI(`experiments/download/zip`, host).search({accession: checkedRows}).toString()}>
-                    Download {checkedRows.length} {checkedRows.length === 1 ? `entry` : `entries`}
-                  </a> :
-                  `Download`
+                  <div>
```
Comment (monicajianu) [inline src/TableContent.js]: I think the conditional rendering could be improved here, so you don't end up with duplication.

**Review by monicajianu — CHANGES_REQUESTED** [review]
It does the job, but it would be better if the tooltip text was passed in as a prop. We want to keep the table generic.

File: src/TableContent.js (hunk 1 above)
Comment (lingyun1010) [reply src/TableContent.js]: I do not think bulk Atlas is going to use download function yet. The question mark is a temporal solution as we discussed, but I agree a props is always good.

File: html/index.html [hunk 3]
```diff
@@ -27,6 +27,12 @@
     resource: 'json/experiments',
     //link to `${host}/${header.resource}/${data[header.link]}/${header.endpoint}`
       species: `homo sapiens`,
+     downloadTooltip: `<ul>
+          <li> Raw filtered count matrix after quantification</li>
+          <li>  Normalised filtered count matrix after quantification</li>
+          <li>  TPM (transcripts per million RNA molecules - only available for SmartSeq)</li>
+          <li>  SDRF file with the experimental design metadata</li>
+        </ul>`,
```
Comment (alfonsomunozpomer) [inline html/index.html:35]: Be consistent using spaces, in this kind of markup it’s better not to have any between the `<li>`  and the text.

File: __test__/ExperimentTable.test.js [hunk 4]
```diff
@@ -16,7 +16,8 @@ describe(`ExperimentTable`, () => {
     resource: `bool`,
     enableDownload: true,
     enableIndex: true,
-    TableCellDiv: TableCellDiv
+    TableCellDiv: TableCellDiv,
+    downloadTooltip: `tooth`
```
Comment (alfonsomunozpomer) [inline __test__/ExperimentTable.test.js]: There are two problems here:
1. The test doesn’t convey the fact that the tooltip content can be HTML (and is rendered as such).
2. Just the word `tooth` is too cryptic. If you fell like a dental comment is in order, be a bit more explicit (e.g. `I can’t think of a good tooltip text here, but I can say that my <b>tooth hurts a lot</b>!). Be conscious of future Atlas developers who might not be able to guess why certain hard-coded values were picked!

**Review by alfonsomunozpomer — CHANGES_REQUESTED** [review]
Two very minor style suggestions. However, coverage has dropped after this PR. Could you fix that, please?

**Review by alfonsomunozpomer — APPROVED** [review]
All good! The new tests are a great addition! Revert the `postTest` phase in `package.json` and squas-merge.

**Pull request merged**